Purchases/Attribution/RCAttributionFetcher.m

```diff
@@ -70,14 +70,16 @@ - (nullable NSString *)identifierForVendor {
 - (void)adClientAttributionDetailsWithCompletionBlock:(RCAttributionDetailsBlock)completionHandler {
     // Should match available platforms in
     // https://developer.apple.com/documentation/iad/adclient?language=objc
-    #if TARGET_OS_IOS
+    #if TARGET_OS_IOS || TARGET_OS_MACCATALYST
```

Comment:
I've added Mac Catalyst, but I didn't remove the comment. I think is useful until this class will be migrated to Swift in order to check again the available platforms.

Reply:
hehe, no problem, I'll update my branch 👍🏼 Swift code is always welcome 🙂 

Well, you're right, it's a double check. IMO, since Mac Catalyst is not so popular it wouldn't make sense to be explicit. So, it's ok for me to use only `iOS`